web/src/test/auto/headless/engine/keyboard/non-positional-rules.js

```diff
@@ -4,8 +4,9 @@ import { ModifierKeyConstants } from '@keymanapp/common-types';
 import { createRequire } from 'module';
 const require = createRequire(import.meta.url);
 
-import { Codes, KeyboardInterface, KeyEvent, MinimalKeymanGlobal, Mock } from '@keymanapp/keyboard-processor';
-import { NodeKeyboardLoader } from '@keymanapp/keyboard-processor/node-keyboard-loader';
+import { Codes, KeyEvent, MinimalKeymanGlobal, Mock } from 'keyman/engine/keyboard';
+import { KeyboardInterface } from 'keyman/engine/js-processor';
```

Comment:
As this references js-processor components (and explicitly references 'rules' in the name!), I'd suggest putting this within an engine/js-processor testing section instead.